src/layouts/DefaultLayout/styles.css

```diff
@@ -0,0 +1,21 @@
+.app-container {
+  display: grid;
+  grid-template-rows: auto 1fr auto; /* header, main, footer */
+  align-items: start;
+  width: 100%;
+}
+
+header {
+  background-color: #3a3a3a;
+  color: black;
+  height: 50px;
+}
+
+main {
+}
+
+footer {
+  background-color: #3a3a3a;
+  color: black;
+  height: 50px;
+}
```

Comment:
Maybe you just haven't styled your header and footer yet, but you'll probably want them to span the entire width of the page when you do.